opal/mca/threads/argobots/threads_argobots_mutex.h

```diff
@@ -141,65 +104,62 @@ static inline void opal_mutex_unlock(opal_mutex_t *m)
     ABT_thread_yield();
 }
 
-/************************************************************************
- *
- * mutex operations (atomic versions)
- *
- ************************************************************************/
+static inline void opal_thread_internal_mutex_destroy(opal_thread_internal_mutex_t *p_mutex)
```

Comment:
If not needed, then don't create it. Set the destructor to NULL.

Reply:
It's confusing, but it's not a destructor. This is a common interface for `opal_thread_internal_mutex_t` (not for the following class).
```c
struct opal_mutex_t {
    opal_object_t super;
    ...
}
```
This defines a necessary "delete" operation even if it's empty for Argobots (Pthreads does `pthread_mutex_destroy()`).

Reply:
Possibly the word "destructor" was confusing. I will change the comment:
```
/* No specific operation to destroy opal_thread_internal_mutex_t. */
```

Reply:
I updated the comments for both Argobots and Qthreads.
```c
    /* No specific operation is needed to destroy opal_thread_internal_mutex_t. */
```